README.md

```diff
@@ -1,5 +1,306 @@
-# Live Streaming API
+<div align="center">
+<h1>Live Streaming API<br/><sub>Control one or more live streaming events on Azure Media Services</sub></h1>
 
-[![Build Status](https://dev.azure.com/literal-life-church/live-streaming-api/_apis/build/status/literal-life-church.live-streaming-api?branchName=develop)](https://dev.azure.com/literal-life-church/live-streaming-api/_build/latest?definitionId=1&branchName=develop)
+<strong>[:one: API](https://github.com/literal-life-church/live-streaming-api/)</strong> |
+[:two: Controller](https://github.com/literal-life-church/live-streaming-controller/) |
+[:three: Player](https://github.com/literal-life-church/stream-switch/)
 
-An interfacing layer to control the status of one or more live streaming events on Azure Media Services.
+```text
+"Streamlining the process to get your events online."
+```
+
+[![Build Status](https://dev.azure.com/literal-life-church/live-streaming-api/_apis/build/status/literal-life-church.live-streaming-api?branchName=develop)](https://dev.azure.com/literal-life-church/live-streaming-api/_build/latest?definitionId=1&branchName=develop) [![Maintainability](https://api.codeclimate.com/v1/badges/094efdcf9b724ad8871f/maintainability)](https://codeclimate.com/github/literal-life-church/live-streaming-api/maintainability) [![Codacy Badge](https://app.codacy.com/project/badge/Grade/60a578f696cb4313849fef9a0589be78)](https://www.codacy.com/gh/literal-life-church/live-streaming-api) [![Postman Documentation](https://img.shields.io/badge/Postman-Documentation%20Available-orange?logo=postman)](https://documenter.getpostman.com/view/3329098/SzzkddRA) ![Live Streaming API Releases](https://img.shields.io/github/v/release/literal-life-church/live-streaming-api?label=Releases)
+
+<hr />
+</div>
+
+## Project Introduction
+
+The Live Streaming API is a subset of a three-part application designed to control the state of one or more live events on [Azure Media Services](https://azure.microsoft.com/en-us/services/media-services/). These parts are:
+
+1. **[Live Streaming API](https://github.com/literal-life-church/live-streaming-api/):** Turn on or off one more live events on Azure Media Services
+1. **[Live Streaming Controller](https://github.com/literal-life-church/live-streaming-controller):** A simple tool for the event broadcaster to interface with the Live Streaming API
+1. **[Stream Switch](https://github.com/literal-life-church/stream-switch):** A front-end, viewer application for viewing one or more live streams on a website
```

Comment:
Codacy found an issue: [[ordered-list-marker-value] Marker should be `3`, was `1`](https://app.codacy.com/gh/literal-life-church/live-streaming-api/pullRequest?prid=5728997)